src/main/resources/config/liquibase/changelog/20210101080900_changelog.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="swaldhauser" id="20210101080900-1">
```

Comment:
please move all changes in this PR to one changeset in one changelog xml and make sure the order is correct.
I will try it out locally on my computer with a dump of the production system and let you know if it works correctly.

The reason is: I would like to avoid too many changelog entries in the database

Reply:
Done in e6818b2bc8d9da65a945fe2da76157737ee1dbd4